Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -505,38 +505,6 @@ class LoggerTests: XCTestCase {
         )
     }
 
-    func testGivenBundlingWithRUMEnabledButRUMMonitorNotRegistered_whenSendingLog_itPrintsWarning() throws {
```

Comment:
Logger does not depend on `RUMMonitor` anymore and we cannot check for its initialisation through integration.